### ISSUE TITLE
Provide example API definition that uses all features of the API.

### ISSUE THREAD

**Issue by atheken** [OAI/OpenAPI-Specification#441]
Although the spec for the swagger definition file appears to be exhaustive, combining all features of the definition schema is very challenging, especially since the example uses for each object definition are not shown in context of their respective YAML/JSON file. It'd be really nice to have at least one definition that showed all available swagger features in one file.

**Comment by webron** [comment]
I think @casualjim has something like it.

We have the pet store (http://petstore.swagger.io). I don't think it's worthwhile to have an example that covers *everything* in a single file. It'd be hard to follow, difficult to maintain, and in general, would just be an awful API :)
